### ISSUE TITLE
Multiple loss training code

### ISSUE THREAD

**Issue by hutuo1213** [fredzzhang/upt#69]
Hi, @fredzzhang :

### if self.training:
            interaction_loss = self.compute_interaction_loss(boxes, bh, bo, logits, prior, targets, pairwise_tokens_x_collated)
            interaction_x_loss = self.compute_interaction_x_loss(boxes, bh, bo, logits, prior, targets, pairwise_tokens_x_collated)
            loss_dict = dict(
                interaction_loss=interaction_loss,
                interaction_x_loss = interaction_x_loss
            )
            return loss_dict

### def _on_each_iteration(self):
        loss_dict = self._state.net(
            *self._state.inputs, targets=self._state.targets)
        if loss_dict['interaction_loss'].isnan():
            raise ValueError(f"The HOI loss is NaN for rank {self._rank}")

        self._state.loss = sum(loss for loss in loss_dict.values())
        self._state.optimizer.zero_grad(set_to_none=True)
        self._state.loss.backward()
        if self.max_norm > 0:
            torch.nn.utils.clip_grad_norm_(self._state.net.parameters(), self.max_norm)
        self._state.optimizer.step()

yaoyaosanqi.

**Comment by fredzzhang** [comment]
Hi @yaoyaosanqi,

It's fairly trivial. How about this
```python
        loss_dict = dict(
            interaction_loss=interaction_loss,
            interaction_x_loss=alpha * interaction_x_loss
        )
```
with `alpha` being the weight on the additional loss.

Fred.

**Comment by hutuo1213** [comment]
Thanks!
